graphql/src/store/query.rs

```diff
@@ -135,7 +191,7 @@ pub fn parse_subgraph_id(entity: &s::ObjectType) -> Option<String> {
         }).and_then(|(_, value)| match value {
             s::Value::String(id) => Some(id),
             _ => None,
-        })
+        }).ok_or(QueryExecutionError::SupgraphIdError(entity.clone().name))
```

Comment:
Do we need to clone the entire entity object type here? Do we have to clone at all here? How about this?
```rust
let entity_name = entity.name.clone();

entity
  .directives()
  ...
  .and_then(|(_, value)| match value {
    s::Value::String(id) => Some(id.clone()),
    _ => None,
  })
  .ok_or(QueryExecutionError::SubgraphIdError(entity_name))
```